Python/instrumentation.c

```diff
@@ -1382,9 +1445,10 @@ initialize_tools(PyCodeObject *code)
 #define NO_LINE -128
 
 static void
-initialize_lines(PyCodeObject *code)
+initialize_lines(PyCodeObject *code, _PyCoLineInstrumentationData *line_data)
```

Comment:
This was so that we could initialize the data and then publish it so any readers would either see no line data or would see the initialized line data.